README.md

```diff
@@ -85,18 +85,18 @@ Results and models are available in the [model zoo](docs/en/model_zoo.md).
 
 ## Installation
 
-Please refer to [install.md](docs/en/install.md) for installation and dataset preparation.
+Please refer to [install.md](https://mmclassification.readthedocs.io/en/latest/install.html) for installation and dataset preparation.
 
 ## Getting Started
-Please see [getting_started.md](docs/en/getting_started.md) for the basic usage of MMClassification. There are also tutorials:
+Please see [Getting Started](https://mmclassification.readthedocs.io/en/latest/getting_started.html) for the basic usage of MMClassification. There are also tutorials:
 
-- [learn about configs](docs/en/tutorials/config.md)
-- [finetuning models](docs/en/tutorials/finetune.md)
-- [adding new dataset](docs/en/tutorials/new_dataset.md)
-- [designing data pipeline](docs/en/tutorials/data_pipeline.md)
-- [adding new modules](docs/en/tutorials/new_modules.md)
-- [customizing schedule](docs/en/tutorials/schedule.md)
-- [customizing runtime settings](docs/en/tutorials/runtime.md)
+- [Learn about Configs](https://mmclassification.readthedocs.io/en/latest/tutorials/config.html)
+- [Fine-tune Models](https://mmclassification.readthedocs.io/en/latest/tutorials/finetune.html)
+- [Adding New Dataset](https://mmclassification.readthedocs.io/en/latest/tutorials/new_dataset.html)
+- [Customizie Data Pipeline](https://mmclassification.readthedocs.io/en/latest/tutorials/data_pipeline.html)
+- [Add New Modules](https://mmclassification.readthedocs.io/en/latest/tutorials/new_modules.html)
+- [Customizie Schedule](https://mmclassification.readthedocs.io/en/latest/tutorials/schedule.html)
+- [Customizie Runtime Settings](https://mmclassification.readthedocs.io/en/latest/tutorials/runtime.html)
 
 Colab tutorials are also provided. To learn about MMClassification Python API, you may preview the notebook [here](https://github.com/open-mmlab/mmclassification/blob/master/docs/en/tutorials/MMClassification_python.ipynb) or directly [run](https://colab.research.google.com/github/open-mmlab/mmclassification/blob/master/docs/en/tutorials/MMClassification_python.ipynb) on Colab.
```

Comment:
Colab tutorials are also provided:

Learn about **MMClassification Python API**: [notebook](xxxx) or  directly [run](xxxx);
Learn about **MMClassification  shell tool**: [notebook](xxxx) or directly [run](xxxx);

may be simpler.